shared/instance.go

```diff
@@ -222,33 +246,18 @@ var KnownInstanceConfigKeys = map[string]func(value string) error{
 	"security.syscalls.intercept.mount.shift":   validate.Optional(validate.IsBool),
 	"security.syscalls.intercept.setxattr":      validate.Optional(validate.IsBool),
 	"security.syscalls.whitelist":               validate.IsAny,
+}
 
-	"snapshots.schedule":         validate.Optional(validate.IsCron([]string{"@hourly", "@daily", "@midnight", "@weekly", "@monthly", "@annually", "@yearly", "@startup"})),
-	"snapshots.schedule.stopped": validate.Optional(validate.IsBool),
-	"snapshots.pattern":          validate.IsAny,
-	"snapshots.expiry": func(value string) error {
-		// Validate expression
-		_, err := GetSnapshotExpiry(time.Time{}, value)
-		return err
-	},
+// InstanceConfigKeysVM is a map of config key to validator. (keys applying to containers AND virtual machines)
```

Comment:
The "keys applying to containers AND virtual machines" bit looks like a copy/paste error here given the var name.